tidy3d/components/medium.py

```diff
@@ -71,6 +71,53 @@ def _eps_model(self, frequency: float) -> complex:
 """ Medium Definitions """
 
 
+class Chi3Spec(Tidy3dBaseModel):
+    """Chi3 medium described by a chi3 nonlinear susceptibility.
```

Comment:
"Specification adding chi3 nonlinear susceptibility to a medium." probably is more accurate.